packages/grid/_modules_/grid/models/api/densityApi.ts

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { DensityTypes, Density } from '../gridOptions';
+
+export interface DensityConfig {
```

Comment:
where is this one used? Why not use the state?

Reply:
You are right, its not use anywhere anymore. I'm removing it. Thanks for the catch ;)